app/main.py

```diff
@@ -1,2 +1,41 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.size = 8
+        self.length = 0
+        self.dict_ = [None for i in range(self.size)]
```

Comment:
If you don't use the loop variable - use `_` as a name.  
It is better to name it what it is - `hash_table`.